EditTextChips/src/main/java/com/novoda/demo/edittextchips/eyeem/EyeemActivity.java

```diff
@@ -0,0 +1,94 @@
+package com.novoda.demo.edittextchips.eyeem;
+
+import com.eyeem.chips.AutocompletePopover;
+import com.eyeem.chips.ChipsEditText;
+import com.novoda.demo.edittextchips.R;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.text.Editable;
+import android.text.TextWatcher;
+
+import java.util.ArrayList;
+
+public class EyeemActivity extends Activity {
+
+    private static final ArrayList<String> TAG_SUGGESTIONS = new ArrayList<String>() {{
```

Comment:
actually it can't - we're enforced by the api to explicitly use `ArrayList` :trollface: https://github.com/novoda/android-demos/pull/23/files/#diff-ea63933cdd671d7c605db38ae61d3371R52